compile.sh

```diff
@@ -145,7 +145,7 @@ fi
 # nproc is a GNU tool, may not be available everywhere, but it is preferable,
 # when it is available
 if hash nproc 2>/dev/null ; then
-  MAXCORES="$(nproc)"
+  MAXCORES="$(nproc)" * 6
```

Comment:
If this is desirable at all

1. Why only here?
2. Why 6?
3. Can't distcc be detected to make this conditional?

Reply:
> If this is desirable at all
> 
>     1. Why only here?

Because that's all I needed for testing locally. I wasn't sure what the maintainers here would ultimately want so that's all I did for now.

> 
>     2. Why 6?

 I just used 6, because I wanted to specify 20 jobs for testing locally.

> 
>     3. Can't distcc be detected to make this conditional?

No.